__tests__/app.test.js

```diff
@@ -0,0 +1,52 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+
```

Comment:
You might be missing the `beforeEach` to seed the test data before every test, and `afterAll` to close the server when the tests are finished